File: 1916-find-center-of-star-graph/1916-find-center-of-star-graph.go
func findCenter(edges [][]int) int {
    set := make(map[int] struct{})
    for i :=0;i<len(edges);i++{
        for j :=0 ;j <len(edges[i]); j++{
            if _ , exists := set[edges[i][j]];exists{
                return edges[i][j]
            }else{
                set[edges[i][j]] = struct{}{}
            }
        }
    }
    return -1
}

File: 1916-find-center-of-star-graph/1916-find-center-of-star-graph.java
class Solution {
    public int findCenter(int[][] edges) {
        HashSet<Integer> set = new HashSet<>();
        for(int i =0;i <edges.length;i++){
            for(int j =0;j <edges[i].length; j++){
                if(set.contains(edges[i][j])){
                    return edges[i][j];
                }else{
                    set.add(edges[i][j]);
                }
            }
        }
        return -1;
    }
}